spec/excon/rails_spec.rb

```diff
@@ -1,7 +1,7 @@
 module Excon
   describe Rails do
     before do
-      stub_request(:any, 'example.com').to_return(body: 'OK')
+      stub_request(:any, /example\.com.*/).to_return(body: 'OK')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.